backend/rates/rates.go

```diff
@@ -37,7 +37,7 @@ import (
 )
 
 // TODO: Unify these with geckoCoin map.
-var coins = []string{"BTC", "LTC", "ETH", "USDT", "LINK", "MKR", "ZRX", "SAI", "DAI", "BAT", "USDC"}
+var coins = []string{"BTC", "LTC", "ETH", "USDT", "WBTC", "PAXG", "DGX", "LINK", "MKR", "ZRX", "SAI", "DAI", "BAT", "USDC"}
```

Comment:
please also extend the `cryptoCompareCoin` and `geckoCoin` maps

Reply:
done